ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -660,7 +660,27 @@ final class Metrics(val registry: MetricRegistry) {
           def push(qualifier: String): Timer = registry.timer(Prefix :+ qualifier :+ "push")
           def slice(qualifier: String): Timer = registry.timer(Prefix :+ qualifier :+ "slice")
           def prune(qualifier: String): Timer = registry.timer(Prefix :+ qualifier :+ "prune")
+
+          val totalTransactionTreesRetrieved: Counter =
+            registry.counter(Prefix :+ "transaction_trees_retrieved_total")
+          val transactionTreeEventsResolvedFromBuffer: Counter =
+            registry.counter(Prefix :+ "transaction_trees_from_buffer")
+
+          val totalFlatTransactionsRetrieved: Counter =
+            registry.counter(Prefix :+ "flat_transactions_retrieved_total")
+          val flatTransactionEventsResolvedFromBuffer: Counter =
+            registry.counter(Prefix :+ "flat_transactions_from_buffer")
         }
+
+        val getTransactionTreesSource: Timer =
+          registry.timer(Prefix :+ "get_transaction_trees_source")
+        val getFlatTransactionsSource: Timer =
+          registry.timer(Prefix :+ "get_flat_transactions_source")
+
+        val transactionTreesBufferSize: Counter =
+          registry.counter(Prefix :+ "transaction_trees_output_buffer_size")
```

Comment:
```suggestion
          registry.counter(Prefix :+ "transaction_trees_buffer_size")
```
In the context of `streams_buffer` there is only one buffer

Reply:
you're right. Done

Reply:
It is still here

Reply:
Sorry, commented in bulk, but disagreed when I checked the code. You have:
* The Akka Streams buffer
* The actual in-memory fan-out buffer
Do you think it is better to not remove it?

Reply:
On a second thought, one might disambiguate from the stream query type that we are not talking about the in-memory fan-out buffer. I'll use your suggestion